src/tree.events.ts

```diff
@@ -41,3 +41,15 @@ export class NodeRenamedEvent extends NodeDestructiveEvent {
     super(node);
   }
 }
+
+export class NodeExpandedEvent extends NodeDestructiveEvent {
```

Comment:
NodeExpandedEvent it is not quite destructive action - it doesn't change amount of children in the tree (in the Tree instance class). The same is for NodeCollapsedEvent. What do you think?